src/renderer/src/views/MigratingProject/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { Button, Container, LinearProgress, Typography, useTheme } from '@mui/material'
+import CheckIcon from '@mui/icons-material/CheckCircleRounded'
+import { Column, Row } from '@renderer/components/LayoutComponents'
+import { Logo, Mapeo5Logo } from '@renderer/components/Logo'
+import { DefaultLayout } from '@renderer/layouts/default'
+import { useEffect, useState } from 'react'
+import { defineMessages, useIntl } from 'react-intl'
+import { useLocation } from 'react-router-dom'
+
+const INTERVAL_DURATION = 250
+
+type LocationState = { observations: number; media: number }
+
+export const MigratingProjectView = () => {
+  const theme = useTheme()
+  const { state } = useLocation()
+  const locationState = state as LocationState
+  const intl = useIntl()
+  const observationsTotal = locationState.observations
+  const mediaTotal = locationState.media
+  const total = observationsTotal + mediaTotal
+  const [observationsMigrated, setObservationsMigrated] = useState(0)
+  const [mediaMigrated, setMediaMigrated] = useState(0)
+  let interval: NodeJS.Timer
+
+  const progress = ((mediaMigrated + observationsMigrated) / total) * 100
+
+  useEffect(() => {
+    interval = setInterval(updateProgress, INTERVAL_DURATION)
+
+    return () => {
+      clearInterval(interval)
+    }
+  }, [])
+
+  useEffect(() => {
+    if (observationsMigrated === observationsTotal && mediaMigrated === mediaTotal) {
+      clearInterval(interval)
+    }
+  }, [observationsMigrated, observationsTotal, mediaMigrated, mediaTotal])
```

Comment:
1. I think this interval needs to be a ref, or a new instance of it will be created on every rerender.

2. I think we can get rid of the second useEffect

```ts
   const interval = useRef<NodeJS.Timer>()

  const progress = ((mediaMigrated + observationsMigrated) / total) * 100

  if(progress >= 100 && interval.current){
    clearInterval(interval.current)
    interval.current = undefined
}

  useEffect(() => {
    if(interval.current)return

    interval.current = setInterval(updateProgress, INTERVAL_DURATION)

    return () => {
      clearInterval(interval.current)
    }
  }, [])
```